src/netcdf.mk

```diff
@@ -32,14 +32,10 @@ define $(PKG)_BUILD
         --enable-netcdf-4 \
         --enable-hdf4 \
         --prefix='$(PREFIX)/$(TARGET)' \
-        CPPFLAGS="-D_DLGS_H" \
+        CPPFLAGS="-DWIN32_LEAN_AND_MEAN -DH4_BUILT_AS_STATIC_LIB=1" \
```

Comment:
Woudl be better if you can hardcode `H4_BUILT_AS_STATIC_LIB` in hdf4 so that all apps using hdf4 can benefit from it.